x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -76,6 +76,8 @@ export const adminMlCapabilities = {
   canCreateTrainedModels: false,
   canDeleteTrainedModels: false,
   canStartStopTrainedModels: false,
+  // Notifications
+  canGetNotifications: false,
```

Comment:
Should this need to be in `userMlCapabilities` rather than an admin only permission??
If it is a replacement for `canGetJobs && canGetDataFrameAnalytics && canGetTrainedModels` these are all user capabilities.

Reply:
Updated in https://github.com/elastic/kibana/pull/143294/commits/ee644b4a877a0fe0f376f0a441c9d21e40d55bd0